provider/equinix/environ.go

```diff
@@ -187,3 +199,245 @@ func (e *environ) Region() (simplestreams.CloudSpec, error) {
 		Endpoint: e.cloud.Endpoint,
 	}, nil
 }
+
+// if values tag and state are left empty it will return all instances
+func (e *environ) getPacketInstancesByTag(tags map[string]string) ([]instances.Instance, error) {
+	toReturn := []instances.Instance{}
```

Comment:
No need for pre-allocating this. Suggest changing to `var toReturn []instances.Instace`